tools/scripts/build_projects.py

```diff
@@ -22,11 +22,11 @@ def parse_input():
 			    formatter_class=argparse.RawTextHelpFormatter)
```

Comment:
these changes to build_projects.py should belong to previous commit.
otherwise, the jsons are really nice and clean, that's gonna be an improvement over what we had so far ;)

Reply:
done